### DISCUSSION TITLE
Pop up

### DISCUSSION THREAD

**Discussion by adarshsingh197** [GrapesJS/grapesjs#5728] [Ideas]
How can i cutomize my own pop up and use it in the canvas and preview, i want to make pop up in the canvas and i want to use it

**Comment by artf** [comment]
You need to create a custom component. Please read the [documentation](https://grapesjs.com/docs/modules/Components.html).

**Comment by adarshsingh197** [comment]
can i make my own pop up in the canvas and hide it after designing and show it in the click of button.
